ouroboros-network/test/Test/Ouroboros/Network/Testnet/Simulation/Node.hs

```diff
@@ -387,6 +390,8 @@ genNodeArgs raps minConnected genLocalRootPeers (ntnAddr, rap) = do
 
   peerSharing <- arbitrary
 
+  ledgerPeers <- arbitrary
```

Comment:
I am a bit suspicious if this is what we want.  The ledger peers need to run so they can be promoted.  Since we have to be careful not to start too many nodes, we could divide `relays` into two parts: local roots and ledger peers.

It would be also good to add a test label which computes how many ledger peers a node promoted as a hot peer - this way we can be sure we are making a difference.

Reply:
done!

Reply:
No, we don't have coverage for diffusion tests, yet

Reply:
Please create an issue for it then.

Reply:
https://github.com/input-output-hk/ouroboros-network/issues/4683